openedx/features/enterprise_support/api.py

```diff
@@ -441,7 +444,9 @@ def get_enterprise_consent_url(request, course_id, user=None, return_to=None, en
     else:
         return_path = reverse(return_to, args=(course_id,))
 
+    enterprise_learner_details = get_enterprise_learner_data(request.site, user)
```

Comment:
Since we're just using the UUID, why aren't we using the `enterprise_customer_uuid_for_request` function created above?